src/main/java/com/example/ezquizbackend/model/Quiz.java

```diff
@@ -0,0 +1,46 @@
+package com.example.ezquizbackend.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+
+@Entity
+// this annotation specifies that the class is an entity and is mapped to a database table. JPA manages this entity
+public class Quiz {
+    @Id
+    // specifies the primary key to the entity
+    @GeneratedValue(strategy  = GenerationType.IDENTITY)
+    // indicates that the primary key value is automatically generated by the database using an identity column
+    private Long id;
+    // field representing the primary key of the entity
+    private String title;
+    // field representing the title of the quiz
+    private String description;
+    // field representing the description of the quiz
+
```

Comment:
In Java, you would typically create constructors here. Since you are using Jakarta, that may not be necessary for now.